doc/api/assert.md

```diff
@@ -1074,6 +1074,12 @@ assert.strictEqual('Hello foobar', 'Hello World!');
 // + 'Hello foobar'
 // - 'Hello World!'
 //          ^
+
+assert.strictEqual(1, 2, 'The input values are not equal');
+// AssertionError [ERR_ASSERTION]: The input values are not equal
+
+assert.strictEqual(1, '1', new TypeError('Expected the input to be identical'));
```

Comment:
The example may be puzzling because lack of strict equality is not necessarily a type error. Maybe this?:

```suggestion
assert.strictEqual(1, '1', new Error('Inputs are not identical'));
```